frontend/institution/forms/forms.css

```diff
@@ -0,0 +1,49 @@
+address-form #address-form {
+  display: grid;
+  grid-template-areas:
+  "street street street"
+  "number neighbourhood neighbourhood"
+  "country state city"
+  "cep cep cep";
+  grid-template-columns: repeat(3, calc(100%/3));
```

Comment:
isn't it the same as `repeat(3, 1fr)` ?

Reply:
Using 1fr causes the form to go out of place in mobile when changing the country from "Brasil" (state and city are select inputs) to something else (state and city are text inputs).